components/shared/view/BasicImage.vue

```diff
@@ -1,32 +1,52 @@
 <template>
-  <b-image
-    :src="src || placeholder"
-    :src-fallback="placeholder"
-    :alt="alt"
-    ratio="1by1"
-    :class="customClass"
-    :rounded="rounded"
-    data-cy="type-image"
-    @error="onImageError">
-    <template #placeholder>
-      <NeoSkeleton full-size no-margin :circle="rounded" />
-    </template>
-  </b-image>
+  <figure class="image-wrapper image is-1by1" :class="customClass">
+    <transition name="fade">
+      <img
+        :src="imageSrc || placeholder"
+        :alt="alt"
+        :class="['has-ratio', { 'is-rounded': rounded }]"
+        @load="onImageLoad"
+        @error="onImageError" />
+    </transition>
+    <transition name="fade">
+      <slot v-if="!loaded" name="placeholder">
+        <NeoSkeleton full-size no-margin :circle="rounded" />
+      </slot>
+    </transition>
+  </figure>
 </template>
 
 <script lang="ts" setup>
 import { NeoSkeleton } from '@kodadot1/brick'
 
 const { $consola } = useNuxtApp()
 const { placeholder } = useTheme()
-defineProps({
-  src: { type: String, default: '' },
-  alt: { type: String, default: 'KodaDot NFT minted multimedia' },
-  customClass: { type: String, default: '' },
-  rounded: Boolean,
-})
+const props = withDefaults(
+  defineProps<{
+    src: string
+    alt: string
+    customClass?: string
+    rounded?: boolean
+  }>(),
+  {
+    src: '',
+    alt: 'KodaDot NFT minted multimedia',
```

Comment:
```suggestion
    alt?: string
    customClass?: string
    rounded?: boolean
  }>(),
  {
    src: '',
    alt: '',
```

Reply:
hmm... alt should be required or have default for better seo